Next overhaul

mrozbarry opened the pull request:
## Summary

Upgrade from my very plain website to something with a little more character.

## Tasks

 - [x] Build up template
 - [ ] Improve mobile support
  - Some elements need to grow
 - [ ] Add links to colleagues, friends, and family
 - [ ] Use Hyperapp to statically render content
 - [ ] Credit https://unsplash.com/photos/xpAma4MLLh4 for image

mrozbarry committed:
Simplify everything

 - Ditch webpack and babel direct usages in favour of parcel
 - Drop resume for now
 - Make website much prettier
 - ???
 - Profit

mrozbarry committed:
Configured firebase app/deployment

mrozbarry committed:
A couple of mobile tweaks

mrozbarry committed:
Added a bit more content for now

jaeh commented on src/apps/index/styles/nav.sass:
```diff
@@ -0,0 +1,87 @@
+  .profile
+    width: 100%
+    height: auto
+    flex: 0
```
```suggestion
```
by removing flex: 0 here we prevent the image from getting distorted on mobile devices (if the other fix above is applied too)

i could not see any problems with this change in the bigger resolution layouts,
if there are, this can be moved into the mobile media query too.

mrozbarry replied:
Which device are you seeing this distortion? I'm using an LG G7 Thinq Android 8 with Chrome, as well as responsive tools on my desktop browser (admittedly, not the most reliable), and I don't see any issue.
Either way, your css suggestions don't do any harm from what I see, so I'll bring them in.

jaeh commented on src/apps/index/styles/nav.sass:
```diff
@@ -0,0 +1,87 @@
+  @media (max-width: $mobile-max-width)
+    position: relative
+    width: 100%
+    height: 100vh
```
```suggestion
    min-height: 100vh
    height: auto
```
firefox & chrome on my machine do not correctly show the full header,
this plus the other fix in this review solved the problem.

jaeh reviewed (commented):
hopefully this will not interrupt your workflow, seemed like the best place to put the changes :)

mrozbarry committed:
Ignore .firebase

mrozbarry commented:
@jaeh now that I brought that in and pull it down, it actually stretches my profile image and distorts it. The `flex: 0` tells the element not to stretch, which keeps the image ratio.

Here is the current, desired image:

![fix-good-proper-ratio](https://user-images.githubusercontent.com/566486/59167389-0663ad00-8aff-11e9-8d1b-fcc08cb65f18.png)

With your fix applied, I get:

![fix-bad-stretch](https://user-images.githubusercontent.com/566486/59167396-12e80580-8aff-11e9-9b92-5f8abf96742f.png)

Which gets stretched downward.

If I'm missing something, let me know, I'd want to resolve this if there's an issue that I'm not seeing.

jaeh commented:
with "mobile devices" above i actually meant my laptop with width manually set to ~400px.
should have clarified :D

interesting, your profile image does not stretch on firefox and chromium on linux _without_ flex: 0,
the stretching only happens on my machine if i _do_ use flex: 0 there.

in desktop view everything is fine either way.

mrozbarry committed:
Ignore public dir

mrozbarry committed:
Fix mobile styling

mrozbarry committed:
Update firebase-tools to 7.x

mrozbarry committed:
More mobile touch-ups

mrozbarry committed:
Add hyperapp-debug testbed app

mrozbarry committed:
Use tailwind instead of my own hand-bombed css

mrozbarry committed:
Removed old index/data

mrozbarry committed:
Another big update

 - Use tailwind
 - Add Felipe to people I've worked with
 - Re-design some layout stuff because of tailwind
